stellarphot/core.py

```diff
@@ -30,6 +30,36 @@
 # https://docs.gammapy.org/dev/_modules/gammapy/analysis/config.html
 
 
+class UnitType(Unit):
+    # Validator for Unit type
+    @classmethod
+    def __get_validators__(cls):
+        yield cls.validate
+
+    @classmethod
+    def validate(cls, v):
+        return Unit(v)
+
+    @classmethod
+    def __modify_schema__(cls, field_schema, field):
+        # Set default values for the schema in case the field doesn't provide them
+        name = "Unit"
+        description = "An astropy unit"
+
+        name = field.name or name
```

Comment:
Is this setting name to a true/false value based on `field.name or name` as a boolean?

Reply:
This sets `name` to `field.name` unless `field.name` evaluates to `False`, in which case `name` is used.